tests/Microsoft.Identity.Test.Common/Core/Mocks/MockHelpers.cs

```diff
@@ -102,9 +102,13 @@ public static Stream GenerateStreamFromString(string s)
             return stream;
         }
 
-        public static HttpResponseMessage CreateResiliencyMessage(HttpStatusCode statusCode)
+        public static HttpResponseMessage CreateServerErrorMessage(HttpStatusCode statusCode, int? retryAfter = null)

```

Comment:
This can also be declared as `int retryAfter = -1)` and change the if statement below. But either way is fine since this is test code.

Reply:
Negative timestamps are scary. I've seen cases where using a negative duration in a backoff retry policy made it accelerate retried, i.e. instead of waiting 2,4,8,16, 32 seconds it waited 2, 1, 0.5 ... seconds. Needless to say it DDOSed the system.